pkg/appfile/validate.go

```diff
@@ -56,7 +56,10 @@ func newValidationProcessContext(wl *Workload, appName, revisionName, ns string)
 	}
 	auxiliaryHooks := []process.AuxiliaryHook{
 		// add more hook funcs here to validate CUE auxiliaries
-		validateAuxiliaryNameUnique(),
+
+		// Disable verify the uniqueness of the fields in the outputs
+		// because users may want to use multiple same type traits on one Component
+		// validateAuxiliaryNameUnique(),
```

Comment:
we can't disable it

Reply:
This is not conflict, you can write a better test case for CUE trait

Reply:
got it